perun-core/src/main/java/cz/metacentrum/perun/core/bl/MembersManagerBl.java

```diff
@@ -1686,4 +1686,15 @@
 	 * @throws SponsorshipDoesNotExistException if the given user is not sponsor of the given member
 	 */
 	void updateSponsorshipValidity(PerunSession sess, Member sponsoredMember, User sponsor, LocalDate newValidity) throws SponsorshipDoesNotExistException;
+
+	/**
+	 * Returns sponsorship, which have validityTo in range [from, to).
+	 * (from is inclusive, to is exclusive).
+	 *
+	 * @param sess session
+	 * @param from lower validityTo bound (inclusive)
```

Comment:
Nit: you can add the info about MIN and MAX value also to this layer.

Reply:
Done.